patches/chrome-installer-mac-signing-commands.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/installer/mac/signing/commands.py b/chrome/installer/mac/signing/commands.py
+index 398a4cafbf4e46fe720f273bf8a4183d07bd4fc9..7e82a335b5c0fe8a6f3dc673d3fe14a48def5228 100644
+--- a/chrome/installer/mac/signing/commands.py
++++ b/chrome/installer/mac/signing/commands.py
+@@ -129,7 +129,7 @@ def macos_version():
+         A list containing one element for each component of the version number,
+         such as [10, 15, 6] and [11, 0].
+     """
+-    return [int(x) for x in platform.mac_ver()[0].split('.')]
++    return [int(x) for x in __import__('signing_helper').BraveGetMacRelease(platform.mac_ver()[0]).split('.')]
```

Comment:
pls rewrite via `brave/chromium_src` file patch if possible. I think it should work here just fine.

raw patches should only be used if a less invasive option is not applicable.

Reply:
Like in 9e0b2ce21f0ace1d2c099557983a92bfd72d66c0? `inline_chromium_src_override` was not possible because `commands.py` is not invoked directly from GN, it is copied into `out/` first. `inline_chromium_src_override` then tries to load `srv/brave/chromium_src/out/...`.